source/module_hamilt_lcao/module_hcontainer/hcontainer.h

```diff
@@ -404,12 +404,17 @@ class HContainer
     /**
      * @brief get sparse_ap_index
      * @return std::vector<std::vector<int>>&
-    */
+     */
     const std::vector<std::vector<int>>& get_sparse_ap_index() const
     {
         return sparse_ap_index;
     }
 
+    const Parallel_Orbitals* get_paraV() const
```

Comment:
Let Hcontainer explicitly depends on Parallel_Orbitals, do we really need this dependency?

Reply:
I think that it depends on the positioning of the Parallel-Orbitals class. If we consider it as a basic class for managing data parallelism information in ABACUS, I think there is no problem returning this member in HContainer.

Reply:
Could you discuss with Daye?

Reply:
After discussed with Daye, we think that it is better to add a function to return the basis number directly.